src/problems/basic_problems.jl

```diff
@@ -32,8 +32,8 @@ Optionally, an initial guess ``u₀`` can be supplied which is used for iterativ
 methods.
 
 ```julia
-LinearProblem{isinplace}(A,x,p=NullParameters();u0=nothing,kwargs...)
-LinearProblem(f::AbstractDiffEqOperator,u0,p=NullParameters();u0=nothing,kwargs...)
+LinearProblem(A,b;u0=nothing,kwargs...)
```

Comment:
why was p removed?

Reply:
If you are passing a matrix, why would this need parameters?
Parameters should only be there when you work with the Operator interface?